source/_components/device_tracker.tile.markdown

```diff
@@ -28,6 +28,7 @@ device_tracker:
     monitored_variables:
       - TILE
       - PHONE
+    show_inactive: false
```

Comment:
Optional configuration arguments should not be in the example. We like to keep the examples as minimal as possible. Please remove this line.

Reply:
Done!